src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+                    onClick={() => addFilter('Front-End', 'filter[]')}
+                >
+                    Front-End
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Back-End', 'filter[]')}
+                >
+                    Back-End
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('QA', 'filter[]')}
+                >
+                    QA
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('DevOps', 'filter[]')}
+                >
+                    DevOps
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Full-Stack', 'filter[]')}
+                >
+                    Full-Stack
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() =>
+                        addFilter(t('description.available'), 'filter[]')
+                    }
+                >
+                    {t('description.available')}
+                </button>
+            </div>
+            <div className={styles['search-active-buttons']}>
+                {searchActiveButtons.map((query) => (
+                    <button
+                        key={query}
+                        className={styles['search-active-button']}
+                        value={query}
+                        onClick={() => removeFilter(query, 'filter[]')}
+                    >
+                        {query}
+                        <FontAwesomeIcon
+                            icon={faTimesCircle}
+                            className={styles['icon']}
+                        />
+                    </button>
+                ))}
+            </div>
+            <div className={styles['employees-items']}>
+                {filterEmployees(employees, 'filter[]')
+                    .filter((emp) => {
+                        const search = searchParams.get('search');
+                        if (!search) {
+                            return true;
+                        }
+                        if (
+                            emp.fullname
+                                .toLowerCase()
+                                .includes(search.toLowerCase()) ||
+                            emp.job
+                                .toLowerCase()
+                                .includes(search.toLowerCase()) ||
+                            emp.availability
+                                .toLowerCase()
+                                .includes(search.toLowerCase())
+                        ) {
+                            return true;
+                        }
```

Comment:
when you use logical operators they will evaluate to boolean values, so there's no need to do
if (logicalExpression) return true else return false
you can just do return logicalExpression
```suggestion
                        return emp.fullname
                                .toLowerCase()
                                .includes(search.toLowerCase()) ||
                            emp.job
                                .toLowerCase()
                                .includes(search.toLowerCase()) ||
                            emp.availability
                                .toLowerCase()
                                .includes(search.toLowerCase())
```